### ISSUE TITLE
Add example project

### ISSUE THREAD

**Issue by jmagnuson** [jmagnuson/fatfs-tiva-cm4f#2]
I have one sitting around somewhere, it just needs to be cleaned up.

**Comment by dollop80** [comment]
Really, do you have one? 
Not clear what should I do on uMDA interrupt. I didn't find uDMA initialization in the code.

**Comment by jmagnuson** [comment]
Yeah, check out the branch I just pushed, [example-project](https://github.com/jmagnuson/fatfs-tiva-cm4f/tree/example-project). I don't have CCS set up at the moment, so you might have to fiddle with some of the project settings to get it working. But the source files, interrupt vector, etc. should all be correct.

It's been a while since I've looked at this code, but the only interrupt that should need handling is [`SDCSSIIntHandler`](https://github.com/jmagnuson/fatfs-tiva-cm4f/blob/example-project/third_party/fatfs/port/mmc-tiva-cm4f.c#L927), and is included in the interrupt vector in [`startup_css.c:102`](https://github.com/jmagnuson/fatfs-tiva-cm4f/blob/0039f8e098f6d44d4193f117d26a0f2ac738a283/startup_ccs.c#L102). That should have been documented in the README, I'll try to remember to do that once I merge the above branch. But check out `sd_util.c` for the driver initialization; that's probably the most useful piece of code for getting started.

Hope that helps!
